.gitignore

```diff
@@ -22,6 +22,9 @@ target/
 /editors/vscode/out/
 /editors/vscode/*.vsix
 
+# JetBrains
```

Comment:
https://github.com/oxc-project/oxc/blob/29edb51c09721088a9878d6c4b2ffd0c2909b0cd/.gitignore#L42-L46

Reply:
The issue is that I don't want to ignore it on my repos
But I will use a local gitignore